script/Common/00_00.txt

```diff
@@ -1522,7 +1527,7 @@ I'm sure that in due time, if it becomes necessary, you will come to understand
 [sha:a1a621036493e71e6287a6cc83e2702d01950a23]{
 -- Page 109, Offset 398.
 -- ―――君の目は、モノの命を軽くしすぎてしまうから」
-With your eyes, it becomes dangerously easy to take a life."
+With your eyes, it becomes dangerously easy to take life."
```

Comment:
take `a` life reads a lot better to me, I assume this change was to make it a bit more generalized but I feel like you can still say `a` life for concepts other than people

Reply:
lives, then!